src/app/GitUI/CommandsDialogs/FormVerify.LostObject.cs

```diff
@@ -59,7 +59,7 @@ private sealed partial class LostObject
             /// <summary>
             /// Diagnostics and object type.
             /// </summary>
-            public string RawType { get; }
+            public string RawType { get; set; }
```

Comment:
❔
```suggestion
            public string RawType { get; private set; }
```

Reply:
Everything done except this one which is not possible because property is updated from FormVerify.
Another "cleaner" solution is much more complex so I thought an exception here was not a big deal for a big perf gain...